lib/API/JSON/v1/Subscribers.php

```diff
@@ -202,7 +202,9 @@ public function sendConfirmationEmail($data = []) {
       if ($this->confirmationEmailMailer->sendConfirmationEmail($subscriber)) {
         return $this->successResponse();
       }
-      return $this->errorResponse();
+      return $this->errorResponse([
+        APIError::NOT_FOUND => __('There was a problem with your sending method. Please check if your sending method is properly configured.', 'mailpoet'),
```

Comment:
```suggestion
        APIError:: UNKNOWN => __('There was a problem with your sending method. Please check if your sending method is properly configured.', 'mailpoet'),
```
@lysyjan tiny detail. All looks good otherwise!

Reply:
@brezocordero Thanks! I fixed it.